src/ios/c_headers/coepicore.h

```diff
@@ -3,96 +3,150 @@
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef generate_tcn(void);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 FFIReturnStruct pass_and_return_struct(const FFIParameterStruct *par);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 int32_t pass_struct(const FFIParameterStruct *par);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef post_report(const char *c_report);
+#endif
 
-CFStringRef record_tcn(const char *c_tcn);
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
+CFStringRef record_tcn(const char *c_tcn, float distance);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 int32_t register_callback(void (*callback)(int32_t, bool, CFStringRef));
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 int32_t register_log_callback(void (*log_callback)(CoreLogMessage));
-
-FFIReturnStruct return_struct(void);
-
-#if defined(TARGET_OS_ANDROID)
-char *rust_greeting(const char *to);
 #endif
 
-#if defined(TARGET_OS_ANDROID)
-char *rust_greeting2(const char *to);
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
+FFIReturnStruct return_struct(void);
 #endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef set_breathlessness_cause(const char *c_cause);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef set_cough_days(uint8_t c_is_set, uint32_t c_days);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef set_cough_status(const char *c_status);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef set_cough_type(const char *c_cough_type);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef set_earliest_symptom_started_days_ago(uint8_t c_is_set, uint32_t c_days);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef set_fever_days(uint8_t c_is_set, uint32_t c_days);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef set_fever_highest_temperature_taken(uint8_t c_is_set, float c_temp);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef set_fever_taken_temperature_spot(const char *c_cause);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef set_fever_taken_temperature_today(uint8_t c_is_set, uint8_t c_taken);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef set_symptom_ids(const char *c_ids);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 int32_t setup_logger(CoreLogLevel level, bool coepi_only);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 CFStringRef submit_symptoms(void);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
 int32_t trigger_callback(const char *my_str);
+#endif
 
+#if (defined(TARGET_OS_IOS) || defined(TARGET_OS_MACOS))
```

Comment:
It would be nice to be able to distinguish methods used for testing/validation (ie. add prefix test_).

Reply:
Open an issue?

Reply:
#126